### DISCUSSION TITLE
Hash changed for 4.0.1 source code tar.gz file?

### DISCUSSION THREAD

**Discussion by mbondurant** [rules-proto-grpc/rules_proto_grpc#246] [Q&A] [answered]
We have been pulling in this file via bazel and this was working on Friday.  It appears today that the file has changed, although the reported date for the release still shows 10/4/2021.  Has there been some corruption of the file?
`WARNING: Download from https://github.com/rules-proto-grpc/rules_proto_grpc/archive/refs/tags/4.0.1.tar.gz failed: class com.google.devtools.build.lib.bazel.repository.downloader.UnrecoverableHttpException Checksum was 8a0775bfd515a2c819d2b1291f430fe7fe71869011934ee2435f6fcd8c31763a but wanted 28724736b7ff49a48cb4b2b8cfa373f89edfcb9e8e492a8d5ab60aa3459314c8`

**Comment by aaliddell** [comment, marked as answer]
See #245

**Reply by mbondurant** [reply]
Thanks for the pointer.  Sorry for the duplicate post as a discussion rather than reviewing the issues first.
